openvm/src/plonk/air_to_plonkish.rs

```diff
@@ -12,163 +14,197 @@ pub fn build_circuit<T>(machine: &SymbolicMachine<T>) -> PlonkCircuit<T, Algebra
 where
     T: FieldElement,
 {
-    let mut circuit = PlonkCircuit::new();
-    let mut temp_id_offset = 0;
+    let mut circuit_builder = CircuitBuilder::<T>::new();
     for constraint in &machine.constraints {
-        air_to_plonkish(&constraint.expr, &mut circuit, &mut temp_id_offset, true);
+        circuit_builder.evaluate_expression(&constraint.expr, true);
     }
 
     for bus_interaction in &machine.bus_interactions {
         add_bus_to_plonk_circuit(
```

Comment:
This could be a method on `CircuitBuilder` ?

Reply:
Agree! Let's do it in a next PR.